src/renderer/pages/player-page.js

```diff
@@ -56,6 +54,14 @@ function renderMedia (state) {
       mediaElement.pause()
     } else if (!state.playing.isPaused && mediaElement.paused) {
       mediaElement.play()
+        .then(() => {
+          dispatch('mediaSuccess')
+        })
+        .catch((err) => {
```

Comment:
Maybe better to comply with: _[use promise then() instead of catch() with callbacks](https://github.com/webtorrent/webtorrent-desktop/pull/1679)_, to prevent we handle exception of the `dispatch` as pointed out by master @feross 😉.

Reply:
Sure! 👍